App/src/main/java/com/google/sps/servlets/Shelter.java

```diff
@@ -10,14 +10,29 @@
     private int ageMin;
     private int ageMax;
     private String description;
-    private static int matchScore = 0;
-    public Shelter(String name, String location, long phoneNum, String description, int ageMin, int ageMax){
+    private int matchScore = 0;
+    private boolean disability;
+    private String gender;
+
+    public Shelter(String name, String location, long phoneNum, String description, int ageMin, int ageMax, boolean disability){
+        this.name = name;
+        this.location = location;
+        this.phoneNum = phoneNum;
+        this.description = "Currently blank";
+        this.ageMin = ageMin;
+        this.ageMax = ageMax;
+        this.disability = disability;
+    }
+    //specific gender shelters
+    public Shelter(String name, String location, long phoneNum, String description, int ageMin, int ageMax, boolean disability, String gender){
```

Comment:
This code is fine, but ask me later about the term "telescoping constructor" and the Builder pattern - there's a way this type of really-long constructor(s) could be made cleaner.